common/etc/nginx/nginx.conf

```diff
@@ -38,6 +38,7 @@ env PROXY_CACHE_VALID_OK;
 env PROXY_CACHE_SLICE_SIZE;
 env PROXY_CACHE_VALID_NOTFOUND;
 env PROXY_CACHE_VALID_FORBIDDEN;
+env PROXY_CACHE_USE_STALE;
```

Comment:
Also be sure to add a validation check in [this file](https://github.com/nginx/nginx-s3-gateway/blob/main/common/docker-entrypoint.d/00-check-for-required-env.sh). I think it would be appropriate to make sure the environment variable is set and it is not blank.